app/report-data/get-claim-level-report-data.js

```diff
@@ -30,8 +30,8 @@ const getClaimLevelReportData = async (startDate, endDate) => {
           "lastUpdated" DESC
       ) AS row_num
     FROM
-      "reportData"
-      ${startDateEndDate}
+    "reportData"
```

Comment:
reportData should be as it was, sorry 
![1bb909a3-1a40-42be-8168-9c944d4eb751_text](https://github.com/DEFRA/ffc-pay-tracking/assets/60177449/471ee90b-5d9d-47fd-84a1-eb8de8868954)